beta-src/src/components/ui/WDDeleteReminder.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import { Box } from "@mui/material";
+import { GameState } from "../../state/interfaces/GameState";
+
+interface WDDeleteReminderProps {
+  notifications: GameState["notifications"];
+}
+
+const WDDeleteReminder: React.FC<WDDeleteReminderProps> = function ({
+  notifications,
+}): React.ReactElement {
+  if (notifications[1] !== undefined) {
+    return <Box sx={notifications[1].style}>{notifications[1].message}</Box>;
+  }
+  // eslint-disable-next-line react/jsx-no-useless-fragment
+  return <></>;
```

Comment:
I think you can try out what Mario is suggesting, another way is to return null (this way you don't need to disable the eslint rule)